templates/download.html

```diff
@@ -1,81 +1,26 @@
 {% extends "base.html" %}
 {% load i18n %}
 {% block content %}
-    <h2>{% trans "Download information about units" %}</h2>
-    <form action='{% url 'download' %}' method='POST'>
-        {% csrf_token %}
-        <div class="row">
-            <div class="col-sm-12">
-            {% if form.download_type.errors %}
-                <div class="form-group has-error has-feedback">
-            {% else %}
-                <div class="form-group">
-            {% endif %}
-                    <label class="control-label">{{ form.fields.download_type.label }}</label>
-                    <select class="form-control" name="download_type">
-                        <option value="" selected>----------</option>
-                        {% for download_type, name in form.fields.download_type.choices %}
-                            <option value="{{ download_type }}">{{ name }}</option>
-                        {% endfor %}
-                    </select>
-                    {% if form.download_type.errors %}
-                        {% for error in form.download_type.errors %}
-                        <span class="help-block"><i class="fa fa-remove"> </i> {{error}}</span>
-                        {% endfor %}
-                    {% endif %}
+    <section class="container text-center">
```

Comment:
Can you add more whitespace so these elements have some breathing room?

<img width="1552" alt="Screen Shot 2022-07-05 at 12 45 57 PM" src="https://user-images.githubusercontent.com/12176173/177386048-9d305ac7-b07d-4678-8cd2-8a0c049276b0.png">
<img width="1552" alt="Screen Shot 2022-07-05 at 12 46 01 PM" src="https://user-images.githubusercontent.com/12176173/177386054-8d362725-3111-4a3e-8ad5-a074fb18778d.png">